static-site/app/blog/parseMarkdown.ts

```diff
@@ -62,7 +78,7 @@ allowedTags.push("text", "textPath", "title", "tref", "tspan", "use", "view", "v
 
 const allowedAttributes = ['href', 'src', 'width', 'height', 'alt'];
 // We add the following Attributes for SVG via https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute
-// Certain values have been excluded here, e.g. "style"
+allowedAttributes.push("style");
```

Comment:
I remember we excluded `style` for a specific reason, I wish we had added as a comment here...
Tracked this down in https://github.com/nextstrain/auspice/pull/834#discussion_r357755220

Reply:
Thanks, I made a mental note to track this down but forgot. Keeping it here with reasoning in comment:

https://github.com/nextstrain/nextstrain.org/blob/ac19562745599f8150795ae7e13c37b99a54ada9/static-site/app/blog/parseMarkdown.ts#L81-L85